.devcontainer/devcontainer.json

```diff
@@ -2,27 +2,36 @@
 // README at: https://github.com/devcontainers/templates/tree/main/src/python
 {
 	"name": "Dash Seedling",
-	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-	"image": "mcr.microsoft.com/devcontainers/python:0-3.10",
-
+	"build": {
+		"dockerfile": "../app/Dockerfile",
+		"context": "../app/"
+	},
 	// Features to add to the dev container. More info: https://containers.dev/features.
-	// "features": {},
 	"features": {
 	"ghcr.io/devcontainers/features/azure-cli:1": {
 			"version": "2.44.1"
 		},
 	"ghcr.io/devcontainers/features/docker-in-docker:2": {}
 	},
 
+	// Configure customizations for VSCode
+	"customizations": {
+		"vscode": {
+			"settings": {
+				"terminal.integrated.profiles.linux": {
```

Comment:
without this the default shell is `sh`. might be cleaner to set `SHELL` in the app dockerfile. thoughts?

Reply:
doesn't work 😕

Reply:
Annoying as if you create a new terminal it works with bash fine

Reply:
@t-young31 can you add me as a collaborator? I've found a fix

Reply:
you're in a group with Maintainer permissions, so you should be able to push to this branch?

Reply:
Works now, no idea why not earlier 🥴